drivers/display/display_video.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2019, Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * display_video is a shim layer allowing to expose a display compatible
+ * interface for video (display/output) devices. The video device needs to
+ * register display_video* methods as display_drv_api callbacks.
+ */
+
+#include <zephyr.h>
+
+#include <display/video.h>
+
+#include <drivers/video.h>
+#include <drivers/display.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(display_video);
```

Comment:
If you do not use LOG_LEVEL in the code just use `LOG_MODULE_REGISTER(display_video, CONFIG_LOG_DEFAULT_LEVEL);` instead.

Reply:
also please change it to CONFIG_DISPLAY_LOG_LEVEL

Reply:
done